scimma_admin/hopskotch_auth/templates/hopskotch_auth/manage_topic.html

```diff
@@ -66,108 +67,55 @@ <h2>Manage Topic</h2>
             </div> 
         </form>
     {% include "hopskotch_auth/messages.html" %}
-    <h2>Added Groups</h2>
+    <h2>Existing Permissions</h2>
     <div class="row mb-3">
         <table class="table" id="added_groups">
             <thead>
                 <th scope="col">Group</th>
                 <th scope="col">Permission</th>
-                <th scope="col">Remove</th>
             </thead>
             <tbody>
                 {% for group in group_list %}
                 <tr scope="row">
                     <td scope="col" class="group_name">{{ group.name }}</td>
-                    <td scope="col" class="perm_fields">{% if group.name == topic_owner %}{% else %}
-                      <div>
-                        Read <input class="readCheck" type="checkbox" {% if group.read %}checked{% endif %}> Write <input class="writeCheck" type="checkbox" {% if group.write %}checked{% endif %}>
-                      </div>
-                      {% endif %}</td>
-                    <td scope="col" class="remove_button">{% if group.name == topic_owner %}Owner{% else %}<button type="button" class="btn btn-danger removeFrom">Remove</button>{% endif %}</td>
+                    <td scope="col" class="permissions">
+                      {% if group.name == topic_owner %}
+                      All (Owner)
+                      {% else %}
+                        {% for perm in group.permissions %}
+                          <span display="inline-block">{{ perm }}&nbsp;<button role="button" style="padding-top: 0; padding-bottom:0;" class="btn btn-sm btn-danger remPerm objectModifier">Remove</button></span> <br>
+                        {% endfor %}
+                      {% endif %}
+                      </td>
                 </tr>
                 {% endfor %}
             </tbody>
         </table>
     </div>
-        <h2>Available Groups</h2>
+        <h2>Available Permissions</h2>
         <div id="cur_permissions">
             <table class="table table-hover" id="avail_table">
                 <thead>
                     <tr>
-                        <th scope="col">Group Name</th>
-                        <th scope="col">Add</th>
+                        <th scope="col">Group</th>
+                        <th scope="col">Permission</th>
```

Comment:
You call them Permissions here, but in the `manage_credential.html` page you called the Operations?

Reply:
Thanks for catching that; fixed in 03c6e9c.